mineinabyss-systems/src/main/kotlin/com/mineinabyss/pvp/PvpGUI.kt

```diff
@@ -0,0 +1,112 @@
+package com.mineinabyss.pvp
+
+import androidx.compose.runtime.*
+import com.mineinabyss.components.playerData
+import com.mineinabyss.guiy.components.Grid
+import com.mineinabyss.guiy.components.Item
+import com.mineinabyss.guiy.components.canvases.Chest
+import com.mineinabyss.guiy.inventory.GuiyOwner
+import com.mineinabyss.guiy.modifiers.Modifier
+import com.mineinabyss.guiy.modifiers.clickable
+import com.mineinabyss.guiy.nodes.InventoryCanvasScope.at
+import com.mineinabyss.idofront.font.NegativeSpace
+import com.mineinabyss.idofront.items.editItemMeta
+import com.mineinabyss.idofront.messaging.error
+import com.mineinabyss.idofront.messaging.success
+import com.mineinabyss.pvp.ToggleIcon.disabled
+import com.mineinabyss.pvp.ToggleIcon.enabled
+import org.bukkit.ChatColor
+import org.bukkit.Material
+import org.bukkit.Sound
+import org.bukkit.entity.Player
+import org.bukkit.inventory.ItemStack
+
+@Composable
+fun GuiyOwner.PvpPrompt(player: Player) {
+    Chest(listOf(player), NegativeSpace.of(18) + "${ChatColor.WHITE}:pvp_menu_toggle:",
+        4, onClose = { reopen() }) {
+        EnablePvp(player)
+        DisablePvp(player)
+        TogglePvpPrompt(player, Modifier.at(8, 3))
+    }
+}
+
+@Composable
+fun EnablePvp(player: Player) {
+    val data = player.playerData
+    Grid(3, 2, Modifier.at(1, 1).clickable
```

Comment:
Don't make the `at` modifier part of the function, this should be specified outside every time to allow reusability